gateway-api/src/lib.rs

```diff
@@ -1,7 +1,15 @@
-pub mod apis;
-
 pub mod duration;
 pub use duration::Duration;
+pub mod standard;
+pub use standard::*;
+
+cfg_if::cfg_if! {
+   if #[cfg(feature = "experimental")] {
+        pub mod experimental;
+    } else {
+
+    }
+}
```

Comment:
Related to above, it seems like we can just drop the cfg_if dep:

```suggestion
#[cfg(feature = "experimental")] {
pub mod experimental;
```

Reply:
Done